frontend/src/js/directives/directives.js

```diff
@@ -32,7 +29,7 @@
                     });
                 }
             };
-            var headerObj = new headerComp();
+            var headerObj = new headerComp(); // eslint-disable-line
```

Comment:
@spyshiv I removed this because it was showing an error (this var is assigned a value but never used) and also this line is used to create an object of a particular function. Problem is solved by removing `var headerObj = new headerComp()` to `new headerComp()` is it ok or do you have any alternate solution to do this?